fennel/internal_lib/to_proto/to_proto.py

```diff
@@ -1104,6 +1113,78 @@ def _renames_to_proto(renames: Optional[Dict[str, str]]) -> Mapping[str, str]:
     return renames
 
 
+def get_sampling_rate(sample: Optional[Union[float, Sample]]):
+    if isinstance(sample, float):
+        if sample < 0 or sample > 1:
```

Comment:
Why to add validation again here? Is the lint failing?

Reply:
yes